### ISSUE TITLE
wiki vs haskell.org page content

### ISSUE THREAD

**Issue by ketzacoatl** [haskell-infra/www.haskell.org#59]
Hello,

While reading about the haskell.org committee, I came across:
* https://www.haskell.org/haskell-org-committee/
* and then also https://wiki.haskell.org/Haskell.org_committee

Which page is the authoritative? Should one just be a link to the other?
I find this type of duplication confusing at best and time-consuming at worst (if I want any confidence in my understanding of the material, I need to read both pages and compare the content).

I'd be happy to help with content updates, but not sure what updates would be welcome.

**Comment by ketzacoatl** [comment]
Is this repo/project maintained?

**Comment by TikhonJelvis** [comment]
Hey, sorry, missed your issue earlier!

I agree that the duplication is confusing. We're currently figuring out what to do with the Wiki in general, so this is something of a transitional state—although things have been moving slowly because everybody on the committee has been extra busy lately.

To me, it seems like the best solution would be to move all the relevant content about the Haskell.org Committee to the official Haskell.org website. In the past, we've gotten feedback that Wiki pages (even if they're edit protected) don't look official. Once we do that, we could replace the Wiki content itself with a link or redirect to the new page.

If you're interested in helping out, you could create a PR on this repo either adding details from the Wiki directly to the existing committee page, or creating another more detailed page with Haskell.org Committee rules/history/etc—whatever reads better. Otherwise I was thinking about creating a PR like that myself, but I won't have time to do that for a few weeks at least, so help is absolutely appreciated!

**Comment by ketzacoatl** [comment]
@TikhonJelvis, that's understandable. Thank you for the guidance there, I'd be happy to at least take a look and see if I can get a PR started.

**Comment by TikhonJelvis** [comment]
Awesome, thanks for looking into it!

**Comment by ketzacoatl** [comment]
@TikhonJelvis it's not perfect, but it's a start. I'd like to see through updating the wiki article to point back at this page, so please review and LMK what additional updates would be best to make here. Thank you for helping to guide and move this along.

**Comment by ketzacoatl** [comment]
That PR has been merged. Now we just need to update the wiki, to replace the existing content with a link to the authoritative page on haskell.org.

**Comment by TikhonJelvis** [comment]
Yep. Heads up that it might take a few days—running into some technical difficulties on my end, and won't be able to deploy the site until they're fixed.

**Comment by TikhonJelvis** [comment]
Finally resolved my technical issues and updated the site. Your changes [should be live now][committee]. We should update the Wiki page too, but I don't think I have a Wiki account, but I'll ask around to find somebody who does.

[committee]: https://www.haskell.org/haskell-org-committee/

**Comment by ketzacoatl** [comment]
@TikhonJelvis I have a wiki account and can make those updates, however, when looking over haskell.org, I could not easily locate the updated page at https://www.haskell.org/haskell-org-committee/. Therefore, I would like to recommend we:

* Update the wiki article to replace the existing content with a link to the authoritative page on haskell.org
* Locate on haskell.org, where the links to the `haskell-org-committee` page will live, and add those links
* Decide if we want to do something about the "if the haskell-org-committee page/url changes, the url in the wiki will be wrong" problem (maybe a comment in the markdown doc about the wiki page?)

**Comment by TikhonJelvis** [comment]
If you have a Wiki account, please feel free to change the Haskell.org committee page to point to `haskell.org/haskell-org-committee`.

There are links to haskell.org/haskell-org-committee from the footer of haskell.org itself as well as the Community and Donations pages. There's also a link to this GitHub repo if people want to make contributions. It sounds like these links weren't clear when you were looking for them—how should we change that?

**Comment by ketzacoatl** [comment]
@TikhonJelvis I tried to do as you suggest on the wiki, but it looks like there is another hurdle to overcome, some new person we need to hunt down and ask for permission to make this update:

![image](https://user-images.githubusercontent.com/10122937/110583738-0c6c3f00-8166-11eb-8ad5-b2a93c6aab3b.png)

> There are links to haskell.org/haskell-org-committee from the footer of haskell.org itself as well as the Community and Donations pages.

I could be crazy, but I'm not seeing it:

![image](https://user-images.githubusercontent.com/10122937/110583856-3cb3dd80-8166-11eb-8e28-f4de34360a9c.png)

**Comment by TikhonJelvis** [comment]
Oh, it's the link where it says "Hosted and managed by Haskell.org, a 501(c)(3) non-profit."

Sounds like the wording is confusing. Would it make more sense if it said "...by the Haskell.org Committee" instead?

**Comment by TikhonJelvis** [comment]
I'm not sure who would have permissions to edit the committee Wiki page; I expect it would be one of the Wiki admins. I'll try to track somebody down for that.

**Comment by gbaz** [comment]
The wiki is not hosted by the haskell.org committee. The committee is just the governing board of the haskell.org nonprofit. The link text is accurate and changing it to say "committee" would be inaccurate.

**Comment by gbaz** [comment]
If any current committee member wants rights on that page, let me know the account name and I can get them added.

**Comment by ketzacoatl** [comment]
I don't have clearance, and it looks like we got stuck on this additional permission check by the wiki admins - @TikhonJelvis can you jump through these hoops with @gbaz so we can finish that first update?

**Comment by ketzacoatl** [comment]
@TikhonJelvis I've started a new PR for a second page that has duplicate / questionable info, please see :point_up: 

Thank you for the feedback, and helping me to consolidate these pages.

**Comment by ketzacoatl** [comment]
@gbaz has unlocked the Donations and Committee articles for updates (thank you!), so I have updated both of those pages:
* https://wiki.haskell.org/Haskell.org_committee
* https://wiki.haskell.org/Donate_to_Haskell.org

#73 still needs updates and to be merged, but that will be easy.

Which pages should we do next? So far I've only focused on pages that I ran into in my own personal needs/research. Are there others that are easy/important/obvious enough to move without a big fuss?

**Comment by ketzacoatl** [comment]
While I believe this is still an issue, should remain open until resolved, and should be resolved, I'm not seeing enough positive feedback or support in making these moves happen for me to continue doing it at this time. I may come back to move more pages in the future, if that were desired and I felt more invigorated, but at least at this time, I need to invest my time elsewhere. I am noting this here so a lack of attention is not considered being neglectful.
